jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -445,13 +447,14 @@ ecma_date_week_day (ecma_number_t time) /**< time value */
 ecma_number_t __attr_always_inline___
 ecma_date_local_tza ()
 {
-  /*
-   * FIXME:
-   *        Get the real system time. ex: localtime_r, gmtime_r, daylight on Linux
-   *        Introduce system macros at first.
-   */
-  TODO ("Implement time functions in jerry-libc.");
-  return ECMA_NUMBER_ZERO;
+  struct timezone tz;
+
+  if (gettimeofday (NULL, &tz) != 0)
+  {
+    return ecma_raise_type_error ("gettimeofday failed");
+  }
+
+  return tz.tz_minuteswest * -60000;
```

Comment:
The timezone is in minutes and we have to convert to milliseconds. I'll give a name for it.